pkg/master/controller.go

```diff
@@ -47,6 +47,7 @@ import (
 type Controller struct {
 	NamespaceRegistry namespace.Registry
 	ServiceRegistry   service.Registry
+	ServiceRest       *service.REST
```

Comment:
> Also @deads2k - if you have suggestions how to improve it.

So we want to go through the "normal" RESTStorage flow to be sure that an IP gets allocated to us.  Any reason we wouldn't use a normal API client and loopback to the API server itself?